tools/esdlcmd/esdl-publish.cpp

```diff
@@ -615,7 +599,7 @@ class EsdlDeleteESDLDefCmd : public EsdlPublishCmdCommon
                     optESDLService.set(arg);
                     break;
                 case 1:
-                    optVersionStr.set(arg);
+                    optDefinitionVersionStr.set(arg);
```

Comment:
Sorry missed this string of comments. I was thinking to deal with this inconsistency in a separate issue, but since we're already here and touched on the code already, I may just do it now.